src/libratbag.c

```diff
@@ -1330,10 +1331,6 @@ ratbag_button_disable(struct ratbag_button *button)
 {
 	struct ratbag_button_action action = {0};
 
-	if (!ratbag_button_has_action_type(button,
-					   RATBAG_BUTTON_ACTION_TYPE_KEY))
-		return RATBAG_ERROR_CAPABILITY;
```

Comment:
Instead of removing this, this should check for `none` action support:
```c


	if (!ratbag_button_has_action_type(button,
					   RATBAG_BUTTON_ACTION_TYPE_NONE))
		return RATBAG_ERROR_CAPABILITY;
```